ext/nnstreamer/tensor_filter/tensor_filter_nnfw.c

```diff
@@ -44,6 +41,9 @@
 #define NNFW_SRCN_BACKEND  "srcn"
 #define NNFW_DEFAULT_BACKEND NNFW_CPU_BACKEND
 
+/** Maximum rank allowed for tensor dimension */
+#define NNFW_TENSOR_RANK_LIMIT 6
```

Comment:
Oh, does NNFW allow tensor rank up to 6 ??

Reply:
[Yes](https://review.tizen.org/gerrit/gitweb?p=platform/core/ml/nnfw.git;a=blob;f=runtime/neurun/api/include/nnfw.h;h=1e18deaab332fb2c05a4a5e3810de3ce6825ab38;hb=refs/heads/tizen#l117)

Reply:
Then, shouldn't it be defined in a header file of nnfw?

Reply:
Anyway, https://lists.fedoraproject.org/pipermail/scm-commits/Week-of-Mon-20150413/1558560.html says "4" is the limit...